docs/reference/api/docker_remote_api.md

```diff
@@ -113,6 +113,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /networks` now supports filtering by `name`, `id` and `type`.
 * `POST /containers/create` now allows you to set the static IPv4 and/or IPv6 address for the container.
 * `POST /networks/(id)/connect` now allows you to set the static IPv4 and/or IPv6 address for the container.
+* `POST /networks/create` now supports restricting external access to the network by setting a `internal` field.
```

Comment:
actually this should be `the` internal field, as @calavera privately suggested to me